gulpfile.js

```diff
@@ -1,17 +1,68 @@
 
 'use strict';
 
-var gulp = require('gulp');
-var modify = require('gulp-modify');
-var cheerio = require('cheerio');
-var concat = require('gulp-concat');
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const htmlExtract = require('gulp-html-extract');
+const lec = require('gulp-line-ending-corrector');
+const stylelint = require('gulp-stylelint');
+const modify = require('gulp-modify');
+const cheerio = require('cheerio');
+const concat = require('gulp-concat');
+
+gulp.task('lint', ['lint:js', 'lint:html', 'lint:css']);
+
+gulp.task('lint:js', function() {
```

Comment:
I'm currently working on vaadin/vaadin-element-skeleton#199 so the lint tasks would be changed soon. 
Let's exclude this change from the current PR.

Reply:
I understand but we would need to update these in all component repos anyway so I thought it wouldn't matter much even if I change these so that the linting is usable here a bit sooner. Do you expect to complete the skeleton lint task changes soon?

So you want me to remove these lint tasks from gulpfile? I'm fine with that, at least I have linting in VS Code due to the linter configs and deps from `vaadin-component-dev-dependencies`.

Reply:
Lint tasks have now been removed from gulpfile.